dictionaries/condensed-strokes.json

```diff
@@ -1568,7 +1567,7 @@
 "TKPW*P/R*/O*/S*/SR*/*E/TPH*/O*/R*": "Grosvenor",
 "TKPW/*ET": "goeth",
 "TKPWA*EUTD": "debated",
-"TKPWAEU/HREU": "gayly",
+"TKPWAEU/TK-LS/HREU": "gayly",
```

Comment:
I'm wondering whether this should perhaps be moved into `dict-en-AU-with-extra-stroke.json` as an alternative for "gaily" outlines eg:

```
"TKPWAEU/HREU/A*U": "gayly"
"TKPWAEUL/HREU/A*U": "gayly"
"TKPWAEUL/KWREU/A*U": "gayly"
```

Internet searching for "gaily vs gayly" garners differing opinions from "gayly" being an "incorrect" spelling of "gaily" to an "old", "uncommon", or "middle English" spelling (which makes me think maybe it wants to go in the AU dictionary...).

What do you think, @didoesdigital ?

Reply:
All good thoughts. I'd be happy to see it in both dictionaries.

Reply:
Okay, done. Added the outlines listed in my previous comment to `dict-en-AU-with-extra-stroke.json`.